src/pages/[nickname]/marking/page.tsx

```diff
@@ -79,81 +134,35 @@ export const UserMarkingPage = withAuth(() => {
     });
   };
 
-  // todo ui 표시
-  if (!token || !myFollowingIdsMap || !myBookmarkIdsMap || !myLikedIdsMap)
-    return null;
+  if (isMarkingListLoading || isClickedMarkingLoading) {
+    return <div>loading..</div>;
+  }
 
   return (
-    <div className="px-4">
-      <BackwardNavigationBar
-        label={
-          <h1 className="title-1 text-grey-900 py-4">
-            {nicknameParams === myNickname ? "내" : nicknameParams} 마킹
-          </h1>
-        }
-      />
-
-      <section className="pb-4 flex flex-col gap-4">
-        {nicknameParams === myNickname &&
-          typeof profile?.tempCnt === "number" &&
-          profile.tempCnt > 0 && (
-            <div className="pt-4">
-              <TemporaryMarkingBar tempCnt={profile.tempCnt} />
-            </div>
-          )}
-
-        <div className="flex w-full justify-end">
-          <SortTypeFilter
-            options={sortTypeOptions}
-            selectedOption={selectedSortType}
-            onSelect={(sortType) => {
-              setSelectedSortType(sortType as Exclude<SortType, "DISTANCE">);
-            }}
-          />
-        </div>
-
-        <MarkingList display="list">
-          {clickedMarking && (
-            <>
-              <MarkingItem
-                {...clickedMarking}
-                onRegionClick={() =>
-                  handleRegionClick({
-                    markingId: clickedMarking.markingId,
-                    lat: clickedMarking.lat,
-                    lng: clickedMarking.lng,
-                  })
-                }
-                isFollowing={myFollowingIdsMap[clickedMarking.userId]}
-                isLiked={myLikedIdsMap[clickedMarking.markingId]}
-                isBookmarked={myBookmarkIdsMap[clickedMarking.markingId]}
-              />
-              <DividerLine axis="row" />
-            </>
-          )}
-          {markingList?.map((marking) => {
-            if (marking.markingId === markingId) return;
-
-            return (
-              <MarkingItem
-                key={marking.markingId}
-                onRegionClick={() =>
-                  handleRegionClick({
-                    markingId: marking.markingId,
-                    lat: marking.lat,
-                    lng: marking.lng,
-                  })
-                }
-                isLiked={myLikedIdsMap[marking.markingId]}
-                isBookmarked={myBookmarkIdsMap[marking.markingId]}
-                isFollowing={myFollowingIdsMap[marking.userId]}
-                {...marking}
-              />
-            );
-          })}
-        </MarkingList>
-        <div className="h-[.125rem]" ref={setNode} />
-      </section>
-    </div>
+    <>
+      <MarkingList display="list">
+        {/* 내 마킹 페이지에서 특정 마커를 클릭한 경우 나타나는 마킹 아이템 */}
+        {clickedMarking && (
+          <MarkingItem onRegionClick={handleRegionClick} {...clickedMarking} />
+        )}
+        {clickedMarking && markingList.length > 0 && <DividerLine axis="row" />}
```

Comment:
DividerLine은 클릭된 마킹이 있다는 전제하에 렌더링되기 때문에, 클릭된 마킹 아이템과 같은 JSX 내부에 배치하는 것이 어떨까요?

```suggestion
        {clickedMarking && (
          <>
            <MarkingItem onRegionClick={handleRegionClick} {...clickedMarking} />
            {markingList.length > 0 && <DividerLine axis="row" />}
          </>
        )}
```

Reply:
오 그거 좋은 생각인 거 같습니다